README.md

```diff
@@ -80,13 +130,43 @@ export const getAnnouncementsByProjectGroup = async value => {
 ```
 This is in addition to the two default "get" functions created. 
 
-## Screenshots
+## Record Transformations
 
-![image](https://user-images.githubusercontent.com/5147486/72138426-7286e780-3352-11ea-8582-f6010de2c390.png)
-Auto-generated `request.js` and `schema.js`
+To make working with records easier, this package includes functions that transform Airtable records after a Read action and before a Create/Update action. That transformation consists of the following: 
 
-## Notes
+#### 1. Javascript-y Column Names
+
+The transformation changes the column names of the record from the column name on Airtable (usually human readable) to the most likely variable name given basic Javascript conventions. 
+
+Examples: 
+- "First Name" -> "firstName"
+- "Is Public?" -> "isPublic"
+
+The current definition of the map from Human Readable to Javascript-y names is: 
+1. Remove Special Characters: `(`, `)`, ` `, `"`, `'`, `?` (more can be added through a PR)
+2. Lowercase Entire String
+3. Split on spaces
+4. Capitalize first character of each word except the first
+5. Combine
+
+#### 2. Accurate Linked Record Column Names
 
-Airtable Schema Downloading Code Credit: https://github.com/cape-io/airtable-schema
+Linked Records on Airtable are usually named something like "Author" or "Project", which would make the corresponding javascript-y name "author" or "project". The most expressive name, however, given how they come back in the API response, would be "authorId" or "projectId". 
+
+Record transformation accounts this, pluralizing it when the record is marked as a one-to-many relationship
```

Comment:
Nit: 
"accounts for this, pluralizing `Id` when the linked record is marked as a one-to-many relationship